requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
This was generated with python 3.10 but the CD uses 3.9, do the packages support 3.9 fully?

Reply:
ah... I'll update to make sure.